charts/consul/templates/api-gateway-controller-deployment.yaml

```diff
@@ -240,6 +241,12 @@ spec:
               fieldPath: metadata.name
         - name: CONSUL_LOGIN_META  
           value: "component=api-gateway-controller,pod=$(NAMESPACE)/$(POD_NAME)"
+        - name: CONSUL_LOGIN_DATACENTER
```

Comment:
I think you might need bats tests in api-gateway-controller-deployment.bats to test the toggle.

Reply:
Added